features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql/h2.sql

```diff
@@ -1967,7 +1969,8 @@ CREATE TABLE IF NOT EXISTS AM_API_CATEGORIES (
   NAME VARCHAR(255),
   DESCRIPTION VARCHAR(1024),
   TENANT_ID INTEGER,
-  UNIQUE (NAME,TENANT_ID),
+  ORGANIZATION VARCHAR(100),
+  UNIQUE (NAME,TENANT_ID,ORGANIZATION),
```

Comment:
Hope we pass tenant domain properly in onPrem case. Hence do we need tenantID in the UNIQUE statement?
Also if we have properly ORGANIZATION populated from the code and the migration, we can drop TENANT_ID column from this.

Reply:
Fixed in https://github.com/wso2/carbon-apimgt/pull/10585/commits/37158abec84c566087fdcf7e29c9fd27849a9f6c